eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/src/cc/codechecker/plugin/views/report/details/BugPathListView.java

```diff
@@ -72,8 +72,11 @@ public void setName(String value) {
         setPartName(value);
     }
 
-    public void clear() {
-        viewer.setInput(null);
+    public void clean(boolean delegate) {
+        if(viewer != null && !delegate) {
+            viewer.setInput(null);
+            viewer.refresh();
+        }
```

Comment:
I don't understand this. If delegate (why is it called delegate?) is false, this method won't do anything. The condition should be outside this function.